VideoEmbedFieldtype.php

```diff
@@ -97,7 +97,7 @@ public function process($data)
             if ($contents = curl_exec($request)) {
                 $contents = json_decode($contents, true);
                 $data['author_url'] = $contents['author_url'];
-                unset($data['duration']); // YouTube does not return duration so lets unset it
+                //unset($data['duration']); // YouTube does not return duration so lets unset it
```

Comment:
I'm not much of a Statamic plugin developer, but just removing that line does nothing to save as seconds in the markdown file. It still saves PT14M58S, but the vue preview displays 898. It is converting correctly, but it never saves.